hugo/content/030-architecture/_index.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Architecture
+remote: https://github.com/gardener/documentation/wiki/Architecture
```

Comment:
section titles are rendered with a very small font for this remote page

Reply:
the reason for this is, that the page uses "####" as markdown. This is indeed very small.
The solution is to fix the remote document and use correct markdown headers like "#" or "##"